src/MgvView.sol

```diff
@@ -60,6 +68,7 @@ contract MgvView is MgvCommon {
   {
     unchecked {
       (MgvStructs.GlobalPacked __global, MgvStructs.LocalPacked __local) = config(olKey);
+      unlockedMarketOnly(__local);
```

Comment:
Can a contract still access the global Mangrove config when at least one market is locked?

Reply:
Yes, they can access it via another offer list. The test `test_offer_list_read_on_reentrancy_succeeds` checks this: https://github.com/mangrovedao/mangrove-core/pull/512/files#diff-dd075adeff468191afc500c88722b7198985b2c71809811e9272fdfed903d160R739-R762

Do you think we should have a view function for reading only the global config?

Reply:
yes -- definitely violates the least surprise principles otherwise

Reply:
Addressed in 3aa854b8d2a4e829c8ffc6a7b21d814a358fbd56